lighthouse-core/test/lib/minification-estimator-test.js

```diff
@@ -10,6 +10,7 @@ const assert = require('assert').strict;
 const {computeCSSTokenLength, computeJSTokenLength} = require('../../lib/minification-estimator.js'); // eslint-disable-line max-len
 
 const angularFullScript = fs.readFileSync(require.resolve('angular/angular.js'), 'utf8');
+const zoneMinifiedScript = fs.readFileSync(`${__dirname}/../../../lighthouse-cli/test/fixtures/dobetterweb/third_party/aggressive-promise-polyfill.js`, 'utf8'); // eslint-disable-line max-len
```

Comment:
can we use this as motivation to move `aggressive-promise-polyfill.js` to the root `third-party/` where it probably should be? :)

Reply:
i looked into it, but

https://github.com/GoogleChrome/lighthouse/blob/13301339637d7d3f6ad708b2759e70c175e85122/lighthouse-cli/test/fixtures/static-server.js#L67

makes that pretty challenging. ideas?

Reply:
we could do another of these

https://github.com/GoogleChrome/lighthouse/blob/13301339637d7d3f6ad708b2759e70c175e85122/lighthouse-cli/test/fixtures/static-server.js#L161-L164

Reply:
plz no this breaks smoke rider

Reply:
> plz no this breaks smoke rider

Can we talk solutions, then? How did smokerider work when we used to serve it out of `node_modules/`?

Reply:
bump

Reply:
> Can we talk solutions, then? How did smokerider work when we used to serve it out of node_modules/?

we skip them. search for `usesExternalResources` in google3 smokerider.

can we check it into `fixtures/third_party` instead, so there is no serving rewiring needed?

Reply:
> so my take is to keep it where it is. but I'd prefer `fixtures/third_party` as a central place for all possible 3p smoke assets

well, those are equally wrong places to put third-party code, so I'm equally fine with them :P

Honestly we should probably delete this file and replace it with a considerably simpler promise stomper (we have a lot more confidence in our isolated context and cached globals than we did when we added this test), but adding it to the unit test is probably going to make it so that never happens :)

Reply:
devtools frontend does this, so it's not without prior art

Reply:
resolution: we handwrite a promisepolyfill for that test.
we get rid of this big one.
maybe use some other minified js for this estimator test.

@paulirish to do